flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/connection/PooledDataSourceFactory.java

```diff
@@ -39,9 +39,10 @@ public static HikariDataSource createPooledDataSource(MySqlSourceConfig sourceCo
 
         String hostName = sourceConfig.getHostname();
         int port = sourceConfig.getPort();
+        String customProperties = sourceConfig.getCustomProperties();
 
         config.setPoolName(CONNECTION_POOL_PREFIX + hostName + ":" + port);
-        config.setJdbcUrl(String.format(JDBC_URL_PATTERN, hostName, port));
+        config.setJdbcUrl(formatJdbcUrlPattern(hostName, port, customProperties));
```

Comment:
Could we pass the jdbc url properties like Kafka connector properties?  
```
jdbc.properties.useInformationSchema=ture
jdbc.properties.characterEncoding=UTF-8
....
```
https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/table/kafka/#properties-bootstrap-servers
In this way we can override all parameters in the JDB_URL